apps/web-twig-demo/assets/scripts/file-uploader-meta-data.ts

```diff
@@ -69,8 +69,8 @@ window.addEventListener('DOMContentLoaded', () => {
   const customEdit = (event) => {
     const key = event.target.closest('li').id;
     const newMeta = toggleMetaData
-      ? { x: 30, y: 30, width: 150, height: 150 }
-      : { x: 22, y: 0, width: 110, height: 100 };
+      ? { x: 30, y: 30, width: 150, height: 150, naturalWidth: 560, naturalHeight: 330 }
+      : { x: 22, y: 0, width: 110, height: 100, naturalWidth: 560, naturalHeight: 330 };
```

Comment:
I'm afraid it may not be quite clear what "natural" means in this context. I suppose it's the _original_ image dimensions, right?

If so, I'd twist the names like this:

- `x` ✅ 
- `y` ✅ 
- `width` ➡️ `cropWidth`
- `height` ➡️ `cropHeight`
- `naturalWidth` ➡️ `width`
- `naturalHeight` ➡️ `height`

I realize it probably means a BC, but I believe it's better in the long run. And backward compatibility should be easy: if `cropWidth` is defined, use `width` and ignore `naturalWidth`, etc.

Reply:
Ok, fixed.